eldritch/test_encounters.py

```diff
@@ -459,6 +510,169 @@ def testLodge5Fail(self):
     self.assertEqual(self.char.clues, 0)
     self.assertEqual(self.char.place.name, "FrenchHill")
 
+  def testLodge6Join(self):
```

Comment:
Feel free to skip tests for Lodge6 if it's exactly the same as Lodge4

Reply:
I kept one just to make sure that my implementation of just calling Lodge4 was going to work.